flox/aggregations.py

```diff
@@ -343,12 +343,61 @@ def _mean_finalize(sum_, count):
 )
 
 
+def _var_combine(array, axis, keepdims=True):
+    def clip_last(array):
+        """Return array except the last element along axis
+        Purely included to tidy up the adj_terms line
+        """
+        not_last = [slice(None, None) for i in range(array.ndim)]
+        not_last[axis[0]] = slice(None, -1)
+        return array[*not_last]
+
+    def clip_first(array):
+        """Return array except the first element along axis
+        Purely included to tidy up the adj_terms line
+        """
+        not_first = [slice(None, None) for i in range(array.ndim)]
+        not_first[axis[0]] = slice(1, None)
+        return array[*not_first]
+
+    assert len(axis) == 1, "Assuming that the combine function is only in one direction at once"
+
+    # Does this double our memory footprint or are they just views?
+    # If there's a huge memory impact, probably better to copy paste array.arrays[1]
+    # in and accept the hit to readability
+    sum_deviations = array.arrays[0]
+    sum_X = array.arrays[1]
+    sum_len = array.arrays[2]
```

Comment:
One minor style comment: you can use "tuple unpacking" here
 ```suggestion
    sum_deviations, sum_X, sum_len = array.arrays
```